src/util/loadExternals.ts

```diff
@@ -29,14 +27,16 @@ class CachedExternal {
   }
 }
 
-/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-function wrappedEval(content: string): any {
-  return eval(content);
+function wrappedEval(content: string): unknown {
+  // in case of a self-invoking wrapper, make sure self is defined
+  // as our context object.
+  return eval(`var self=this;\n${content}`);
 }
 
 function getWindowCache(): CachedExternals {
-  window.__isolatedExternalsCache = window.__isolatedExternalsCache || {};
-  return window.__isolatedExternalsCache;
+  const wind = window as Window;
```

Comment:
Is this change to make it more Typescript friendly?

Reply:
It's a little weird because there's an eslint validation warning around not using the Window interface (defined above), but this is taking advantage of [TypeScript's interface merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces), so `window` already is part of the Window interface.